### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by gongweibao**

### CODE REVIEW

File: doc/design/profiler.md [hunk 1]
```diff
@@ -12,22 +12,22 @@ for i in xrange(M):  # M is  the iteration number
     op.run();
 ```
 
-In summary, the proflier should have following features:
+In summary, the profiler should have following features:
```
Comment (abhinavarora) [inline doc/design/profiler.md]: have following features -> _have **the** following features_

File: doc/design/profiler.md [hunk 2]
```diff
@@ -12,22 +12,22 @@ for i in xrange(M):  # M is  the iteration number
...
 
-- records time span in loop.
+- records time span in a loop.
```
Comment (abhinavarora) [inline doc/design/profiler.md]: It would be good to capitalize the first letter of each of these points

File: doc/design/profiler.md [hunk 3]
```diff
@@ -12,22 +12,22 @@ for i in xrange(M):  # M is  the iteration number
...
+But how to record the time for the mixed C++ and CUDA program?  There many C++ APIs to get the current calendar time in the host program. But for GPU, the CUDA kernels may be executed concurrently if they are in different [streams](http://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#streams) and the CUDA kernels are asynchronous with the host program if there is no the synchronous after the CUDA kernels. CUDA provides [event](http://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#events) to monitor the device and performs accurate timing. Inspired by PyTorch and CUDA event, we also design and apply the events to record the timeline. Then summarize and present statistics based on these events.
```
Comment (abhinavarora) [inline doc/design/profiler.md]: the  mixed C++ and CUDA program -> _**a** mixed C++ and CUDA program_

File: doc/design/profiler.md [hunk 4]
```diff
@@ -12,22 +12,22 @@ for i in xrange(M):  # M is  the iteration number
...
 
-The overall flow is shown as the following figure.
+The overall flow is shown in the following figure.
 
 <img src="./images/profiler.png" align="center"/><br/>
 
 ### Event
 
-In above work flow, a pair of events are needed before and after the piece of code to collect time. So the event has a flag to mark whether it is a starting event or an ending event. Except this two kinds of event, sometime, a only marker with a text message is needed, for example, a marker to specify the profiling start or end. There are three kinds of event:
+In above work flow, a pair of events is needed before and after the piece of code to collect time. So the event has a flag to mark whether it is a starting event or an ending event. Except this two kinds of event, sometimes, an only marker with a text message is needed, for example, a marker to specify the profiling start or end. There are three kinds of event:
```
Comment (abhinavarora) [inline doc/design/profiler.md]: Except this two kinds -> _Except **for these** two kinds_

File: doc/design/profiler.md [hunk 5]
```diff
@@ -12,22 +12,22 @@ for i in xrange(M):  # M is  the iteration number
...
 ```
 
-In summary, the proflier should have following features:
+In summary, the profiler should have the following features:
 
-- records time span in loop.
-- supports nested time span.
-- supports multiple threads/multiple GPUs.
-- supports to be enabled and disabled by users.
+- Record time span in a loop.
```
Comment (kuke) [inline doc/design/profiler.md]: Should be `Records` due to `profiler`. The same below.

Comment (gongweibao) [reply doc/design/profiler.md]: Thanks! Done.

**Comment by luotao1** [comment]
感谢您给PaddlePaddle贡献文档。由于文档已迁移至[FluidDoc](https://github.com/PaddlePaddle/FluidDoc) repo，因此关闭您的PR，欢迎您向[FluidDoc](https://github.com/PaddlePaddle/FluidDoc) Repo贡献文档。
Thanks for contributing to PaddlePaddle! Since documents have been moved to [FluidDoc](https://github.com/PaddlePaddle/FluidDoc) repo, we close this PR. Welcome to contribute to [FluidDoc](https://github.com/PaddlePaddle/FluidDoc) repo.

**Pull request closed without merging**